solr/test-framework/src/java/org/apache/solr/BaseDistributedSearchTestCase.java

```diff
@@ -189,40 +134,19 @@ public static void clearSolrDisableShardsWhitelist() throws Exception {
   }
 
   private static String getHostContextSuitableForServletContext() {
-    String ctx = System.getProperty("hostContext", "/solr");
-    if (ctx == null || ctx.isEmpty()) ctx = "/solr";
-    if (ctx.endsWith("/")) ctx = ctx.substring(0, ctx.length() - 1);
-    if (!ctx.startsWith("/")) ctx = "/" + ctx;
+    String ctx = "/solr";
+
     return ctx;
```

Comment:
```suggestion
    return "/solr";
```

Or just inline this thing!  It's just a constant string after all.